package.json

```diff
@@ -37,32 +37,49 @@
   },
   "homepage": "https://github.com/bento-platform/Bento-Charts#readme",
   "peerDependencies": {
+    "leaflet": "^1.9.4",
     "react": ">=16.0.0",
     "react-dom": ">=14.0.0",
+    "react-leaflet": "^4.2.1",
```

Comment:
If it is not too much work it would be worth it I think.
An option could be to make this repo a monorepo, where bento_charts is a pure recharts package, and we add a separate package bento_maps that uses bento_charts. But this could become hard to maintain.

Reply:
If we take the multi-package monorepo route, a tool like [Lerna](https://lerna.js.org/) could be useful